modules/jaggery-apps/app-store-web/extensions/assets/webapp/pages/subscriptions.jag

```diff
@@ -78,7 +79,14 @@ require('/modules/store.js').exec(function (ctx) {
 
     }
 
-
+	//Recent app rating.
+	for (var i = 0; i < recentAssets.length; i++) {
+		if (JSON.parse(social.getRating("webapp:" + recentAssets[i].id))) {
+			recentAssets[i].rating = {average: Math.floor(JSON.parse(social.getRating("webapp:" + recentAssets[i].id)).rating)};
```

Comment:
can change to for performance improvement. Here JSON parse seems to call twice for the same purpose.
var asset = JSON.parse(social.getRating("webapp:" + recentAssets[i].id));
if(asset != null) {
ecentAssets[i].rating = asset.rating;
} ...